src/test/locales/locales.test.js

```diff
@@ -13,3 +13,14 @@ describe('locales/en', () => {
     expect(localeEn.messages).toBeDefined();
   });
 });
+
+describe('coherence test between locales files', () => {
+  const itKeys = Object.keys(localeIt.messages);
+  const enKeys = Object.keys(localeIt.messages);
+  it('same number of keys in locales files', () => {
+    expect(itKeys.length).toEqual(enKeys.length);
+  });
+  it('same key values in each file', () => {
+    expect(itKeys).toEqual(enKeys);
```

Comment:
Object.keys does not guarantee the order. You can either implement this as a forEach, or calling .sort() before comparing the two arrays